src/gfx/module.ts

```diff
@@ -606,11 +608,32 @@ module Shumway.GFX {
       // A complete polyfill of currentTransform isn't feasible: we want to only use it if it gives
       // us a meaningful value. That we can only get if the platform gives us any means at all to
       // get that value. Gecko does so in the form of mozCurrentTransform, most engines don't.
-      if (!('currentTransform' in CanvasRenderingContext2D.prototype) &&
-        'mozCurrentTransform' in CanvasRenderingContext2D.prototype) {
-        Object.defineProperty(CanvasRenderingContext2D.prototype, 'currentTransform', {
-          get: mozPolyfillCurrentTransform
-        });
+      // For Chrome, at least return whatever transform was set using setTransform to ensure
+      // clipping works in our 2D backend.
+      if (!hasCurrentTransform) {
```

Comment:
This will not work if any of the transformation functions like `translate`, or `scale`, `rotate` are called on the context. If you don't plan to support this at least override them with stubs that throw assertions.